pyproject.toml

```diff
@@ -21,6 +21,8 @@ dependencies = [
     "pint-xarray",
     "matplotlib",
     "access-config-utils",
+    "experiment-runner",
+    "payu", # Extra dependency of experiment-runner, not handle correctly in conda
```

Comment:
What happens if you don't add payu? Do you want to create an issue and add the details of why `payu` needs to be here

(Is there a separate payu-dev that packages experiment-runner/generator?)